.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ workflows:
       - orb-tools/review:
           filters: *filters
       - shellcheck/check:
-          exclude: SC2148,SC2038,SC2086,SC2002,SC2016,SC1090
+          exclude: SC2148,SC2038,SC2086,SC2002,SC2016,SC1090,SC2046,SC2129
```

Comment:
Do we still want these additions?

Reply:
I just removed them and `shellcheck` passed